tests/unittests/lib.collections.res

```diff
@@ -310,6 +310,15 @@ Art :string
 
 [A r t u r o] :block 
 [[1 2 3] [4 5 6] [7 8]] :block 
+[directory wofilerld] :block 
+[D: User user .arturo] :block 
+[ usr bin] :block 
+[directory wofilerld] :block 
+[D: User user .arturo] :block 
+[ usr bin] :block 
+[directory wofilerld] :block 
+[D: User user .arturo] :block 
+[ usr bin] :block 
```

Comment:
```suggestion
[directory wofilerld] :block 
[ usr bin] :block 
[directory wofilerld] :block 
[ usr bin] :block 
[directory wofilerld] :block 
[ usr bin] :block 
```

And - hopefully - this would be the expected output... (I so hope I haven't messed up... lol)